test/test_accumulator.cpp

```diff
@@ -20,12 +20,12 @@
 #endif
 #endif
 
-#include <rcppmath/rolling_mean_accumulator.hpp>
-
-#include <gtest/gtest.h>
 #include <cmath>
 #include <memory>
 
+#include "gtest/gtest.h"
+
+#include <rcppmath/rolling_mean_accumulator.hpp>
```

Comment:
Shouldn't this use the double-quotes instead of the brackets?

Reply:
Now using double-quotes (e2bd343)